numba/misc/dump_style.py

```diff
@@ -0,0 +1,76 @@
+
+import numba.core.config
+from pygments.styles.default import DefaultStyle
+from pygments.styles.manni import ManniStyle
+from pygments.styles.monokai import MonokaiStyle
+from pygments.styles.native import NativeStyle
+
+from pygments.lexer import RegexLexer, include, bygroups, words
+from pygments.token import Text, Name, String,  Punctuation, Keyword, \
+    Operator, Number
+
+from pygments.style import Style
```

Comment:
I think it'd be good to have a `try..except ImportError` around this to catch the case where Numba extension writers might try and import this module for use in their code but not have `pygments` installed (as it's only an optional requirement for Numba).  Thinking something like:
https://github.com/numba/numba/pull/5333/files#diff-5de309739e66ef02bd05362a66f0caa1R191-R201